### PR TITLE
Update LVL to 1.0.28 spec version

### PR DESCRIPTION
**Pull request by mark-lunarg**

### CODE REVIEW

**Comment by MarkY-LunarG** [comment]
LGTM

**Comment by courtney-g** [comment]
:+1:

File: generator.py [hunk 1]
```diff
@@ -3142,6 +3143,8 @@ def genCmd(self, cmdinfo, name):
             for param in params:
                 len = self.getLen(param)
                 if len:
+                    if '::' in len:
```
Comment (stroyan) [inline generator.py]: This appears redundant because getLen already does this replacement.

File: generator.py [hunk 2]
```diff
@@ -2580,6 +2580,7 @@ def makeThreadUseBlock(self, cmd, functionprefix):
                     else:
                         # externsync can list members to synchronize
                         for member in externsync.split(","):
+                            member = str(member).replace("::", "->", 1)
```
Comment (stroyan) [inline generator.py]: If there is more than one '::' in the string we will probably want to replace them all.  So we won't want the max replacement count of 1.

File: generator.py [hunk 3]
```diff
@@ -3256,6 +3259,8 @@ def getLen(self, param):
                 result = len.split(',')[0]
             else:
                 result = len
+            if '::' in result:
+                result = str(result).replace('::', '->', 1)
```
Comment (stroyan) [inline generator.py]: The 'if' test is not needed.  The replace will only affect when that is true.
This replacement count of 1 could also be removed.

File: generator.py (hunk 2 above)
Comment (mark-lunarg) [inline generator.py]: Done

File: generator.py (hunk 1 above)
Comment (mark-lunarg) [inline generator.py]: Thanks Mike!

File: generator.py (hunk 3 above)
Comment (mark-lunarg) [inline generator.py]: Done.

**Comment by mark-lunarg** [comment]
Pushed to Master.  Thanks for the review help.

**Pull request closed without merging**